### DISCUSSION TITLE
npx not working in VS code

### DISCUSSION THREAD

**Discussion by Cory7717** [expo/expo#29460] [Archived: Questions]
I've installed node.js on my Windows system and when I try to run npx create-expo-app, it returns "The term 'npm' is not recognized..."  See below for the exact error.  Any insight?

npm : The term 'npm' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path 
was included, verify that the path is correct and try again.
At line:1 char:1
+ npm create-expo-app@latest
+ ~~~
    + CategoryInfo          : ObjectNotFound: (npm:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

**Comment by alanfrancis442** [comment]
fix1:
open C:\Users\${your_user_name}\AppData\Roaming\
If there is no folder called npm create one
and add the folder path to the environment variables as listed in fix 2

fix2:
open the file location where node is installed
copy the file path
open system environment variables
click on path
click on edit
click on new
and past the copied path
then save the changes
